### PR TITLE
Closes #474

### LINKED ISSUE #474: Add [UIView] to UIStackView
Create an extension to add an array of views to `UIStackView`.

### Suggested names:
- `addArrangedSubviews(_views: [UIView])`
- `appendArrangedSubviews(_views: [UIView])`

### PR DESCRIPTION
**Pull request by omaralbeik**
Add `addArrangedSubviews(_ views: [UIView])` and `removeArrangedSubviews()` extensions to `UIStackView`.

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [x] New extensions are written in Swift 4.
- [x] New extensions support iOS 8.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+.
- [x] I have added tests for new extensions, and they passed.
- [x] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [x] All extensions are declared as **public**.
- [x] I have added a [changelog](https://github.com/SwifterSwift/SwifterSwift/blob/master/CHANGELOG_GUIDELINES.md) entry describing my changes.

### CODE REVIEW

**Comment by LucianoPAlmeida** [comment]
Hey @omaralbeik :))
I think we need to investigate this `testDebounce` implementation, I've seen it failing in many other PR's but normally it works properly and it's just rebuilt and it fixes. But I think there's something not right with this implementation. It definetly something to take a look.

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,19 +19,19 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
...
 - **Optional**:
   - Added `isNilOrEmpty` property to check whether an optional is nil or empty collection.
 - **UIWindow**:
   - Added `switchRootViewController` method to switch root view controller with animation. [#494](https://github.com/SwifterSwift/SwifterSwift/pull/494) by [omaralbeik](https://github.com/omaralbeik).
-
 - **Sequence**
   - added `containsDuplicates()` to check whether a sequence contains duplicates. [#496](https://github.com/SwifterSwift/SwifterSwift/pull/496) by [@vyax](https://github.com/vyax).
   - Added `single(where:)` to get the only element of a sequence that matches a given condition. [#483](https://github.com/SwifterSwift/SwifterSwift/pull/483) by [andlang](https://github.com/andlang).
+- **UIStackView**:
+  - Added `addArrangedSubviews(_ views: )` to add an array of views to the end of the arrangedSubviews array. [#474](https://github.com/SwifterSwift/SwifterSwift/issues/474) by [omaralbeik](https://github.com/omaralbeik).
```
Comment (LucianoPAlmeida) [inline CHANGELOG.md]: This should be the number of PR I think :)

**Comment by SD10** [comment]
That error message from `xcode_summary` is so clear and beautiful. I love it already @LucianoPAlmeida 😍

**Comment by BennX** [comment]
LGTM @omaralbeik 👍

**Pull request merged**